lapce-ui/src/editor/tab.rs

```diff
@@ -706,8 +707,10 @@ impl TabRectRenderer for TabRect {
             );
         };
 
-        if is_pristine || self.close_rect.contains(mouse_pos) {
-            if self.rect.contains(mouse_pos) || is_active_tab {
```

Comment:
I took the liberty of removing one check here because `close_rect` is a subset of `rect`.